### ISSUE TITLE
Unresolve custom methods and properties in window

### ISSUE THREAD

**Issue by vgpechenkin** [tomblachut/svelte-intellij#222]
Hi, Tomasz.
Thank you so much for you working on the plugin. 
I have some problems. The plugin doesn't resolve custom methods and properties. 
I using svelte on Game UI and c++ expose methods and properties in window global object.

global.d.ts
```
declare global {
	interface Window {
		ContractService: ContractService;
	}
}
```

Using 
![image](https://user-images.githubusercontent.com/1455274/121009117-7499e380-c79c-11eb-9cd5-715670e99ea7.png)

**Comment by tomblachut** [comment]
Hey, last time I've checked Svelte had builtin list of allowed globals, did something change in that area?

**Comment by vgpechenkin** [comment]
Hey. 
I don't know about builtin list of allowed globals. 
I create an empty test project and 2 svelte files.
App and Component. 
![image](https://user-images.githubusercontent.com/1455274/121517792-6e07a800-c9f8-11eb-9ae4-45d11d78367f.png)

**Comment by tomblachut** [comment]
I'm not sure if it should compile, does it?  in Svelte you can't access whatever from window object as global (without `window.`) and I assumed same can be said about `window.` access

**Comment by vgpechenkin** [comment]
Hi @tomblachut. 
I think this problem relate to https://youtrack.jetbrains.com/issue/WEB-51050.
When I moved importing svelte file to ts It solves the problem

**Comment by tomblachut** [comment]
Ok, thank you.

**Comment by tomblachut** [comment]
Hi @vgpechenkin, could you check if this super secret attachment fixes the problem? (2021.2 required) 
[svelte-intellij-0.21.0-alpha.zip](https://github.com/tomblachut/svelte-intellij/files/6865091/svelte-intellij-0.21.0-alpha.zip)

**Comment by vgpechenkin** [comment]
Hi @tomblachut.
Yes, This version fixes the bug.

**Comment by tomblachut** [comment]
I'm glad it works. Thanks for testing.

Fixed in #232
